### PR TITLE
 Build metadata for Fable 2 REPL

### PR DESCRIPTION
**Pull request by alfonsogarciacaro**
@ncave Only for reference, please don't merge if there's a better alternative. These are the changes I had to do to to compile the metadata for Fable 2 REPL, besides updating the dependencies, target netcoreapp2.0 instead as the REPL couldn't read netcoreapp2.1 assemblies (I also downgraded the dotnet SDK to 2.1.200 though I'm not sure if this is strictly needed).

### CODE REVIEW

**Comment by ncave** [comment]
@alfonsogarciacaro I have no problems using netcoreapp2.1 assemblies in the bench, in fact I'm only using 2.1. But whatever works, it's not that important, in the sense that using either 2.0 or 2.1 is fine.

File: fcs/build.fsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ let isMono = false
 #endif
 
 
-let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.200"
+let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.401"
```
Comment (ncave) [inline fcs/build.fsx]: @alfonsogarciacaro 2.1.402 is the preferred version as everything below has vulnerability (not applicable here but still).

File: fcs/fcs-export/fcs-export.fsproj [hunk 2]
```diff
@@ -15,5 +16,7 @@
     <PackageReference Include="Dotnet.ProjInfo.Matthid" Version="*" />
     <PackageReference Include="Fable.Core" Version="2.0.0-alpha-012" />
     <PackageReference Include="Fable.Import.Browser" Version="1.1.3" />
+    <PackageReference Include="Fable.Core" Version="1.*" />
```
Comment (ncave) [inline fcs/fcs-export/fcs-export.fsproj]: @alfonsogarciacaro Duplicate references

**Comment by ncave** [comment]
@alfonsogarciacaro Sorry, just realized you're probably not intentionally modifying this PR, just fixing after the rebase from 25.0.1. Please ignore.

**Comment by alfonsogarciacaro** [comment]
Actually, it'd be great if this could be merged so I don't have to deal with so many forks :) But I've also added a reference to the Repl Lib and this only works with my directory. Not sure what's the best way to do this, I could push a Nuget package of the Repl.Lib just to restore it in fcs-export.fsproj, but it would be a bit cumbersome.

**Comment by ncave** [comment]
@alfonsogarciacaro I know, fork of a fork of FCS which is itself a fork of a fork of MS F# :)
Too many forks, not enough philosophers :)
I can merge it if you can remove the Repl Lib reference, so the build would work for everybody. But frankly it's just a base tool to generate metadata, it's meant to be changed by adding your own references, so just reference your fork (of a fork of a fork of a fork).

**Comment by alfonsogarciacaro** [comment]
Actually, after thinking a bit about it, probably the best solution is to add a custom fcs-export.fsproj to the REPL repository. I will send a different PR to parameterize the project directory [in this line](https://github.com/ncave/FSharp.Compiler.Service/blob/9c4f6600114656119a2473744da161506a46089f/fcs/build.fsx#L130).

**Pull request closed without merging**